lib/modules/opengraph_builder.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+# This class is meant to be used as a wrapper for
+# Opengraph meta-tags within HTML markup.
+#
+class OpengraphBuilder
+  attr_accessor :data
+
+  # Create an instance e.g. og = OpengraphBuilder.new
+
+  # The initializer accepts a hash with the starting structure for example...
+  #
+  # og = OpengraphBuilder.new(
+  #   { og: { title: 'my title' }, twitter: { card: 'my_card' } }
+  # )
```

Comment:
I would remove this comment as it is already clear from the code how to initialize this class

Reply:
I removed the first line of this comment but think it's good to keep line 11-15 just so that it's obvious that the hash should be two-levels in depth beginning with the prefix (e.g. og or twitter)